metrique/tests/kitchen-sink.rs

```diff
@@ -117,6 +134,34 @@ struct IgnoredField {
     _b: IAmNotAMetric,
 }
 
+#[metrics(rename_all = "snake_case")]
+#[derive(Default)]
+struct Snakify {
+    #[metrics(flatten)]
+    nested: Nested,
+}
+
+#[metrics(rename_all = "kebab-case")]
+#[derive(Default)]
+struct Kebabify {
+    #[metrics(flatten)]
+    nested: Nested,
+}
+
+#[metrics(subfield, rename_all = "PascalCase")]
```

Comment:
oh...I didn't realize we supported `rename_all` for subfields. I wonder if we should make this a warning. `subfields` should probably always use the name scheme of their parent.